internal/github/checkPendingCI.go

```diff
@@ -136,7 +136,7 @@ func checkTeamMemberActivity(ctx context.Context, client *github.Client, owner,
 	}
 
 	for _, comment := range comments {
-		slogs.Logr.Info("Checking comment by team member", "user", comment.User.GetLogin(), "created_at", comment.CreatedAt.Format(time.RFC3339), "PR", prNumber, "repository", repo)
+		slogs.Logr.Info("Checking comment by:", "user", comment.User.GetLogin(), "created_at", comment.CreatedAt.Format(time.RFC3339), "PR", prNumber, "repository", repo)
```

Comment:
I actually preferred the original version of this logline I think, because with slog it doesn't really make sense to put a semicolon and have a bunch of key-values follow it

Reply:
I originally had "team member" which is not actually true, since its pulling all comments, including those who are not employees. But, I did change this to "user" and removed the colon.